vulnerabilities/importers/osv.py

```diff
@@ -188,24 +200,26 @@ def parse_advisory_data_v2(
     )
 
 
-def extract_fixed_versions(fixed_range) -> Iterable[str]:
+def extract_introduced_and_fixed(ranges) -> Tuple[List[str], List[str]]:
```

Comment:
This don't look right, you're yielding a list of tuples and not a tuple of lists.